src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -145,14 +146,16 @@ Okta, Azure, and OneLogin have New Relic apps for both the original user model a
 
 **Recommended**: Download the full list of existing original user model users before choosing to import users. This will be a useful resource and serve as a backup, if you need it. 
 
-After downloading your original user model users, you can upload all users or just some of them. This step will create user records on the New Relic One user model. In a later step, you’ll be able to transition these users' assets. 
+After downloading your original user model users, you can upload all users or just some of them. We recommend reviewing this list of users and removing any users you don't want migrated (for example, people who no longer work at your organization). 
 
-The new user record that's created has the same login credentials: there is no need to reset passwords. If a user has a pending email verification status (pending being verified), that will also be transitioned over. 
+When you reupload your list of users and complete this step, it will create user records on the New Relic One user model. In a later step, you’ll be able to transition these users' assets. 
+
+The new user records created for your users have the same login credentials: there's no need to reset passwords. If a user has a pending email verification status (pending being verified), that will also be transitioned over. 
 
 Important tips: 
 
 * Ensure the new users' email addresses match their original user record email addresses, including matching exact case. We use email addresses as the key value to match users and, in a later step, to transition their user-associated assets. 
-* Once you complete this step and create new user records, we highly recommend completing the migration process without delay. If you don't complete the steps to migrate assets and delete the original user record, a user may see two user records when logging in (see [login screenshot from Step 1](#page1)) or else may be missing assets they expect to see (like dashboards). 
+* Once you complete this step and create new user records, we highly recommend completing the migration process without delay. If you don't complete the steps to migrate assets and delete the original user record, a user may have two user records associated with the same login (see [login screenshot from Step 1](#page1)) or else may be missing assets they expect to see (like dashboards). 
```

Comment:
@zuluecho9, you're saying some readers might stop at step 5? If so, perhaps you could be more explicit and tell readers to continue with the remaining steps (like a pharmacist telling you to finish your antibiotics).